roles/os_firewall/tasks/firewall/iptables.yml

```diff
@@ -21,7 +21,7 @@
   ignore_errors: yes
 
 - name: Install iptables packages
-  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
```

Comment:
Filed https://bugzilla.redhat.com/show_bug.cgi?id=1324914
Found https://bugzilla.redhat.com/show_bug.cgi?id=1096752

I looked at their spec file and I cannot figure out what's wrong with the package.